blank.html

```diff
@@ -1,18 +1,15 @@
 <!DOCTYPE html>
 <html lang="en" id="site" class="nojs">
 <head>
-<!--[if lt IE 9]>
-<script src="static/dist/bower_components/html5shiv/dist/html5shiv.min.js"></script>
-<![endif]-->
-
     <title>blank page</title>
     <meta charset="utf-8" />
     <meta name="viewport" content="initial-scale=1.0">
     <link rel="shortcut icon" href="/static/dist/img/favicon.ico">
 
+    <link rel="stylesheet" type="text/css" href="static/dist/css/vendor.min.css"/>
```

Comment:
Making sure `normalize.css` and other vendor-specific CSS are included before the sitewide css.